man/timesyncd.conf.xml

```diff
@@ -74,6 +74,19 @@
         is used instead.</para></listitem>
       </varlistentry>
 
+      <varlistentry>
+        <term><varname>NetworkTimeSecurity=</varname></term>
+        <listitem><para>A space-separated list of Network Time Security (NTS) server host
+        names or IP addresses to be used. Network Time Security (NTS), a mechanism for
```

Comment:
```suggestion
        names or IP addresses to be used. Network Time Security (NTS) is a mechanism for
```
Otherwise this sentence doesn't have a verb.